src/main/lrsql/system.clj

```diff
@@ -9,9 +9,13 @@
 (defn system
   "Return a lrsql system with configuration specified by the `profile`
    keyword."
```

Comment:
Make sure to document the new `conf-overrides` arg in the docstring.

Reply:
done